src/services/BookService.tsx

```diff
@@ -1,5 +1,5 @@
 import client from "./client";
 
-export const fetchAllBooks = async () => client.get("/books");
+export const fetchAllBooks = async (params: any) => client.get(`/books?page=${params}`);
```

Comment:
Please avoid using `any` type

```suggestion
export const fetchAllBooks = async (params: string) => client.get(`/books?page=${params}`);
```